samples/S8.Flow/S8.Flow.Api/wwwroot/js/modals.js

```
@@ -0,0 +1,819 @@
+    }
+  }
+
+  async fetchEntityDetails(entityId, entityType) {
+    switch (entityType) {
+      case 'device':
+        return await window.flowApi.getDevice(entityId);
+      case 'sensor':
+        return await window.flowApi.getSensor(entityId);
+      case 'reading':
+        // There isn't a direct reading-by-id route; return recent as a placeholder
+        const page = await window.flowApi.getReadings({ size: 1 });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)